### PR TITLE
Added netatmo api

### PR DESCRIPTION
**Pull request by bbuie**
Added netatmo as an api.

### CODE REVIEW

File: dist/ng-cordova-oauth.js [hunk 1]
```diff
@@ -1067,7 +1067,9 @@
     'oauth.untappd',
     'oauth.dribble',
     'oauth.pocket',
-    'oauth.mercadolibre'])
+    'oauth.mercadolibre',
+    'oauth.xing',
```
Comment (bbuie) [inline dist/ng-cordova-oauth.js]: I'm not sure where oauth.xing came from, it may be on the master branch?

File: src/oauth.js [hunk 2]
```diff
@@ -41,7 +41,8 @@
     'oauth.dribble',
     'oauth.pocket',
     'oauth.mercadolibre',
-    'oauth.xing'])
+    'oauth.xing',
```
Comment (bbuie) [inline src/oauth.js]: I'm not sure where oauth.xing came from. I may have started on the master branch or something.

**Comment by nraboy** [comment]
Can you explain a few things?:

1. I believe we had Xing support.  Did it get lost somewhere and you added it back in?
2. Have you tested both Netatmo and Xing since you touched both?

Once you confirm that everything works, I'll merge the PR.

Thanks,

**Comment by bbuie** [comment]
@nraboy I didn't touch anything with Xing. I'm a little confused why that is in there. I can take it out if you like. 

I have tested with Netatmo.

**Comment by bbuie** [comment]
@nraboy it looks like Xing was only added to the dist file because it was already in the uncompiled file but never compiled correctly (I guess).

**Comment by nraboy** [comment]
Xing was contributed in PR #238 by @visiongeist.

When you created your development branch, did you use the already existing development upstream branch or did you create one based on the master branch.  If you created a development branch based on the master, then you are going to wipe out a lot of changes that were done on the development.

Please let me know what you did.

Thanks,

**Comment by bbuie** [comment]
@nraboy I'm pretty sure I branched off of development and am now pushing back to development. If I branched off master, you'd see a whole lot more differences between my code and the development branch. If you look at #238, you'll see that there were no changes to the "dist" files. That leads me to believe they just forgot to compile.

**Comment by nraboy** [comment]
Ah you know what, you're right!  I overlooked that.

Looks good, I will merge the PR.  Thank you for contributing to the project :-)

**Pull request merged**